### PR TITLE
Improve auto showing

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
This is an improvement on https://github.com/neo4j/neo4j-browser/pull/1285.

See http://better_auto_show.surge.sh

### CODE REVIEW

File: src/browser/modules/Frame/FrameTitlebar.tsx [hunk 1]
```diff
@@ -90,10 +90,8 @@ import arrayHasItems from 'shared/utils/array-has-items'
 import { stringifyMod } from 'services/utils'
 import Monaco, { MonacoHandles } from '../Editor/Monaco'
 import { Bus } from 'suber'
-import FeatureToggle from '../FeatureToggle/FeatureToggle'
-import { reusableFrame } from 'shared/modules/experimentalFeatures/experimentalFeaturesDuck'
 import { addFavorite } from 'shared/modules/favorites/favoritesDuck'
-import uuid from 'uuid'
+import { useCustomBlur } from 'browser-components/SavedScripts/hooks'
```
Comment (HerrEmil) [inline src/browser/modules/Frame/FrameTitlebar.tsx:94]: unused import?

**Review by HerrEmil — COMMENTED** [review]
When I try the demo, multiline queries are folded nicely, expend as expected when clicking, but then they stay expanded without ever collapsing again. That is unexpected to me, at least when minimizing the frame I expect the editor to fold to one line again

File: src/browser/modules/Frame/FrameTitlebar.tsx (hunk 1 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Frame/FrameTitlebar.tsx:94]: Thanks!

**Comment by OskarDamkjaer** [comment]
I've made the collapse button as you requested and removed the unused import!

**Review by HerrEmil — APPROVED** [review]
🎉

**Pull request merged**